dotnetsdk.rb

```diff
@@ -0,0 +1,36 @@
+require "formula"
+
+class Dotnetsdk < Formula
+  homepage "https://www.github.com/aspnet/Home"
+  version "1.0.0-beta3"
```

Comment:
We wanted to have a solution here so that we don't have to update the formula every release. Can we think about fixing this or updating this automatically ?